e2e-tests/tfm/cypress/e2e/journeys/case-amendments/add-bank-decision/add-bank-decision-proceed.spec.js

```diff
@@ -99,7 +115,7 @@ context('Amendments underwriting - add banks decision - proceed', () => {
     cy.clickContinueButton();
     cy.url().should('contain', 'cover-end-date');
 
-    cy.completeDateFormFields({ idPrefix: 'amendment--cover-end-date', date: tomorrow.date });
+    cy.completeDateFormFields({ idPrefix: 'amendment--cover-end-date', date: twoDays.date });
```

Comment:
is there a reason this has changed to two days instead of tomorrow? Ideally we should keep these bits of the test the same as before

Reply:
Because once every 4 years the test will fail if it is tomorrow and it won't if we change it to the day after tomorrow becuse of how leap years work

Reply:
@Zainzzkk I left a comment asking for a change here to `twoDays` instead. Reason is: 
- The test here is just checking that updating the cover end date on the date field forms shows the updated one in the summary. Whether or not it's changed to `tomorrow` or `twoDays` doesn't matter for checking this. 
- If we have `tomorrow` we have some complicated leap year issues that will occur once a year (left in my other comment). Making it 2 days in the future instead of tomorrow means these will never occur, and is the quickest, cleanest fix.

Reply:
I'll revert if you are keen to leave it though, keen to get this merged ASAP

Reply:
I've redone the maths and I think it actually might be safe as usually tomorrow will give 25 months, and then the edge case is on Feb 28th on a leap year the default would be 24 months, but then as both are end of month, we add 1 and get 25 months, so updated tenor should still be constant.

Have reverted.